airbyte-webapp/src/config/utils.ts

```diff
@@ -1,4 +1,13 @@
-import { ConnectionScheduleDataBasicSchedule } from "core/request/AirbyteClient";
+import { ConnectionScheduleData } from "core/request/AirbyteClient";
 
-export const getFrequencyType = (schedule?: ConnectionScheduleDataBasicSchedule) =>
```

Comment:
Not sure why this is in config/utils instead of core/analytics, but that problem was already there. If this was an analytics util it would be along the lines of `getFrequencyFromScheduleData`

Reply:
Are you suggesting a move/rename?

Reply:
I think so since we're already here.